src/phoenix/experimental/evals/models/openai.py

```diff
@@ -151,11 +217,77 @@ def _build_messages(
     def _verbose_generation_info(self) -> str:
         return f"OpenAI invocation parameters: {self.public_invocation_params}"
 
-    def _generate(self, prompt: str, **kwargs: Dict[str, Any]) -> str:
+    def generate(
+        self, prompts: List[str], instruction: Optional[str] = None, num_consumers: int = 20
+    ) -> List[str]:
+        return asyncio.run(self._generate_async(prompts, instruction, num_consumers=num_consumers))
+
+    async def _generate_async(
```

Comment:
Can we rename this function more clearly? Since we also have `async def _generace(...)` on line 286, it's getting harder to remember which is which.